src/main/java/org/apache/maven/plugin/gpg/GpgVersionParser.java

```diff
@@ -90,7 +90,7 @@ public GpgVersion getGpgVersion()
     static class GpgVersionConsumer
         implements StreamConsumer
     {
-        private final Pattern gpgVersionPattern = Pattern.compile( "gpg \\(GnuPG\\) .+" );
+        private final Pattern gpgVersionPattern = Pattern.compile( "gpg \\(GnuPG.+\\) .+" );
```

Comment:
This will fail for everything else, but your case. It should probably be non-greedy.

Reply:
Of course, sorry, fixed that with the next (forced push) commit and added a test case.